lib/app_navigator.dart

```diff
@@ -63,19 +63,19 @@ class AppNavigatorImpl extends AppNavigator {
 
   @override
   void navigateToHomeScreen({required BuildContext context}) {
-    context.replace('$_routePathRootScreen$_routePathHomeScreen');
+    context.replace('/$_routePathHomeScreen');
```

Comment:
I am a bit confused here as `const _routePathRootScreen = '/';` already defined. So, is replacing `_routePathRootScreen` with `/` make any difference ? @Tuubz

Reply:
Sorry for not explaining. No there's no difference, it's the same 🙏 

Initially, I was thinking to re-factor this:

 `context.push('/$_routePathFormScreen/$surveyId');`

to this:

`context.push('$_routePathRootScreen$_routePathFormScreen$_routePathRootScreen$surveyId');`

But this makes it unnecessary hard to read, so I decided to hardcode it, what do you think? 🙏

Reply:
the confusion is clear now, I am ok with it.